fuzzing/src/visitors/glyphvisitor.h

```diff
@@ -18,21 +18,19 @@
 
 #include "utils/utils.h"
 
-#include <boost/core/noncopyable.hpp>
-
-
 namespace freetype {
 
 
   class GlyphVisitor
-    : private boost::noncopyable
   {
   public:
 
 
     virtual
     ~GlyphVisitor() = default;
 
+    GlyphVisitor(const GlyphVisitor&) = delete;
+    GlyphVisitor& operator=(const GlyphVisitor&) = delete;
```

Comment:
Now that these are declared, you'll also need to declare the default constructor (as default). (Always have https://howardhinnant.github.io/smf.jpg handy.) I think there are a few other classes which also need this. (boost::noncopyable got away with this by declaring constructors in the super class.)

Reply:
Thanks, added explicit default constructors where needed. 

> (Always have https://howardhinnant.github.io/smf.jpg handy.)

😄

Updated coding style (space after brackets) as well.